admin/SetLanguageFrequencies

```diff
@@ -13,11 +13,11 @@ use Try::Tiny;
 
 use constant MAX_TOP_LANGUAGES => 20;
 
-# Always include the six official UN languages and the "Multiple languages"
+# Always include the six official UN languages, the "Multiple languages" and the "No linguistic content" / "No lyrics"
 # pseudo language.
 #
 use constant PRESELECTED_LANGUAGES => qw(
-    ara eng fra spa rus zho mul
```

Comment:
This would set no linguistic content into the top "frequently used" list, which I understand from the ticket you don't think is needed?

Reply:
Exacly y'oure righjt!

Reply:
Reverted. :)